### PR TITLE
add checks for format exceptions in csharp wrapper

### PR DESCRIPTION
**Pull request by MathieuMorrissette**

### CODE REVIEW

File: wrappers/csharp/Utils.cs [hunk 1]
```diff
@@ -11,7 +11,14 @@ public static byte[] StringToByteArray(string data)
                 return null;
             }
 
-            return Encoding.UTF8.GetBytes(data);
+            try
+            {
+                return Encoding.UTF8.GetBytes(data);
```
Comment (ekse) [inline wrappers/csharp/Utils.cs]: I don't think we need this try-catch. The exceptions listed for GetBytes are `ArgumentNullException`, which we already check for, and `EncoderFallbackException`, but we do not set it `EncoderExceptionFallback`.

https://docs.microsoft.com/en-us/dotnet/api/system.text.encoding.getbytes?view=netframework-4.8

File: wrappers/csharp/Utils.cs [hunk 2]
```diff
@@ -20,8 +27,15 @@ public static string ToBase64String(byte[] bytes)
             {
                 return null;
             }
-
-            return Convert.ToBase64String(bytes);
+            
+            try
```
Comment (ekse) [inline wrappers/csharp/Utils.cs]: The only listed exception is `ArgumentNullException` which we check for, I don't think we need this try-catch.

Comment (MathieuMorrissette) [reply wrappers/csharp/Utils.cs]: Fixed

File: wrappers/csharp/Utils.cs (hunk 1 above)
Comment (MathieuMorrissette) [reply wrappers/csharp/Utils.cs]: Fixed

**Pull request merged**